akka-actor-typed/src/main/scala/akka/actor/typed/internal/StashBufferImpl.scala

```diff
@@ -94,6 +96,12 @@ import akka.util.ConstantFun
   override def unstashAll(ctx: javadsl.ActorContext[T], behavior: Behavior[T]): Behavior[T] =
     unstashAll(ctx.asScala, behavior)
 
+  override private[akka] def unstashAll(): im.Seq[T] =
+    new Iterator[T] {
+      override def hasNext: Boolean = StashBufferImpl.this.nonEmpty
+      override def next(): T = StashBufferImpl.this.dropHead()
+    }.to[im.Seq]
```

Comment:
this should return collection with number of elements that was there when it was called, because more might be added when iterating (risk of infinite loop), easy way is to add `.take(size).to[im.Seq]`